src/middleware/get-payload.ts

```diff
@@ -1,19 +1,19 @@
-module.exports = getPayload;
+import { IncomingMessage } from "http";
+import { Payload } from "../types";
 
-function getPayload(request) {
+export function getPayload(request: IncomingMessage): Promise<Payload> {
   // If request.body already exists we can stop here
   // See https://github.com/octokit/webhooks.js/pull/23
-  if (request.body) {
-    return Promise.resolve(request.body);
-  }
+  // if (request.body) {
+  //   return Promise.resolve(request.body);
+  // }
```

Comment:
This change is causing the error with the server test you see right now.

How about we do this?
```suggestion
  // @ts-ignore
  if (request.body) return Promise.resolve(request.body);
```

I think it's more explicit then altering the `request` type